tests/export_back_compat_test.py

```diff
@@ -142,23 +141,17 @@ def test_ducc_fft(self):
     def func(x):
       return lax.fft(x, fft_type="fft", fft_lengths=(4,))
 
-    # An old lowering, with ducc_fft. We keep it for 6 months.
-    data = self.load_testdata(cpu_ducc_fft.data_2023_03_17)
-    if jaxlib_version <= (0, 4, 20):
-      expect_current_custom_calls = ["dynamic_ducc_fft"]
-    else:
-      # We have changed the lowering for fft since we saved this data.
-      # FFT no longer lowers to a custom call.
-      expect_current_custom_calls = []
+    # TODO(slebedev): Remove this test and the corresponding custom calls
```

Comment:
I think the time line is six months, actually. (Google tracking bug b/311175955 for removing this custom call in about one month from now.)

Reply:
We had a chat about this with @gnecula, and George told me that we start counting from the day we bumped the minimum jaxlib to the version where the custom call is no longer emitted. Wdyt?

Reply:
I disagree with that. I think it starts from the day we released a version that no longer emits the custom call.

Otherwise these deprecation timelines are going to be way too long. We only can bump the minimum quarterly at best.

Reply:
In this particular case, is there a big cost to us to keep the jaxlib support for the custom call alive for a few more months? We can remove the lowering code.

It is technically possible that somebody used the old lowering until today, when we forced them to upgrade the jaxlib, because the choice of lowering was conditional on the jaxlib version. We could have been more proactive to bump the minimum jaxlib version sooner to force users to use the new lowering sooner.

Reply:
The minimum jaxlib version is not something we can bump freely, I'm afraid. Only once a quarter roughly at the moment.